src/test/java/de/unistuttgart/iste/gits/flashcard_service/test_utils/TestUtils.java

```diff
@@ -0,0 +1,102 @@
+package de.unistuttgart.iste.gits.flashcard_service.test_utils;
+
+import de.unistuttgart.iste.gits.common.resource_markdown.ResourceMarkdownEntity;
+import de.unistuttgart.iste.gits.flashcard_service.persistence.dao.FlashcardEntity;
+import de.unistuttgart.iste.gits.flashcard_service.persistence.dao.FlashcardSetEntity;
+import de.unistuttgart.iste.gits.flashcard_service.persistence.dao.FlashcardSideEntity;
+import de.unistuttgart.iste.gits.flashcard_service.persistence.repository.FlashcardSetRepository;
+import jakarta.transaction.Transactional;
+import org.springframework.boot.test.context.TestComponent;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+
+@Component
+public class TestUtils {
+    /**
+     * Helper method which creates some flashcard sets and saves them to the repository.
+     * @param repo The repository to save the flashcard sets to.
+     * @return Returns the created flashcard sets.
+     */
+    @Transactional(Transactional.TxType.REQUIRES_NEW) // Required so the data is committed to the repo before the
+    // rest of the test method which calls this method is executed.
+    public List<FlashcardSetEntity> populateFlashcardSetRepository(FlashcardSetRepository repo) {
```

Comment:
Interesting way to create test data. I personally prefer builders, but this works too 👍